docs/guides/qunasys-quri-chemistry.ipynb

```diff
@@ -122,180 +125,300 @@
-    "| :----------------- | :------------- | :------------------------------------------------------------------------------------------ | :------- | :------ | :-------------------------------------------------------------- |\n",
-    "| mole               | json           | The molecule setting. See the table for setting molecules above                             | Yes      |         | The molecule setting. See the table for setting molecules above |\n",
-    "| ansatz             | str            | The name of the ansatz you want to use. Only “DoubleExcitation” and “UCCSD” are supported. `DoubleExcitation` is highly recommended to obtain a stable result, especially for systems larger than eight qubits   | Yes      |         | “DoubleExcitation”                                              |\n",
-    "| state_prep_method  | str            | The way you want to prepare the circuit parameter of your ansatz. Only CCSD is allowed | No       | “CCSD\"  | “CCSD”                                                          |\n",
-    "| ansatz_setting     | str            | Specific setting you want to configure your ansatz with                                     | No       | None    | See the following examples                                          |\n",
-    "| qsci_setting       | dict[str, int] | Specify the number of shots and the size of the subspace. See the following section for details                                                       | Yes      |         | See the QSCI setting table                                      |\n",
-    "| mitigation_setting | json           | Settings for configuring error mitigation                                                   | No       | None    | See the mitigation section below                                |"
+    "#### Symmetry post selection\n",
+    "\n",
+    "These keys control the symmetry post selection: `particle_number_symmetry` and `spin_symmetry`. Setting `particle_number_symmetry` to True selects those states that conserve the total electron numbers.  Setting `spin_symmetry` to True selects those states that conserve the $s_z$ quantum number. Currently only $s_z = 0$ is supported. The default is:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "mitigation_setting = {\n",
+    "    \"particle_number_symmetry\": True,\n",
+    "    \"spin_symmetry\": True,\n",
+    "}"
    ]
   },
   {
    "cell_type": "markdown",
-   "id": "06d15114-0779-42b9-8239-a7049a9cfbf1",
    "metadata": {},
    "source": [
-    "#### QSCI algorithm parameter\n",
+    "#### Configuration recovery\n",
     "\n",
-    "The parameters for setting the QSCI algorithm is summarized in this table.\n",
+    "Instead of the symmetry post selection, you can also choose to do configuration recovery. To turn it on, set it to `True`, and optionally customize it with a dictionary containing the following parameters:\n",
```

Comment:
```suggestion
    "Instead of the symmetry post-selection, you can also choose to do configuration recovery. To turn it on, set it to `True`, and optionally customize it with a dictionary containing the following parameters:\n",
```